### PR TITLE
Fix pipeline failures in personalizer

### PR DESCRIPTION
**Pull request by sharathmalladi**
# Description

Fix build pipeline failures.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java [hunk 1]
```diff
@@ -164,8 +165,11 @@ public Flux<ByteBuffer> getAsync(Boolean signed, Context context) {
     public InputStream get(Boolean signed) {
         Iterator<ByteBufferBackedInputStream> iterator =
                 getAsync(signed).map(ByteBufferBackedInputStream::new).toStream().iterator();
+        abstract class ModelReference implements Enumeration<InputStream> {
+        }
+
         Enumeration<InputStream> enumeration =
-                new Enumeration<InputStream>() {
+                new ModelReference() {
```
Comment (samvaity) [inline sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java:172]: We usually shouldn't be modifying Generated classes. Why was this required?

File: pom.xml [hunk 2]
```diff
@@ -904,6 +914,7 @@
         <module>sdk/eventhubs</module>
         <module>sdk/extendedlocation</module>
         <module>sdk/formrecognizer</module>
+        <module>sdk/personalizer</module>
```
Comment (JimSuplizio) [inline pom.xml:917]: Please add this module entry in alphabetical order, it should be after the sdk/peering entry

File: pom.xml [hunk 3]
```diff
@@ -80,6 +80,9 @@
         <!-- Form Recognizer -->
         <module>sdk/formrecognizer/azure-ai-formrecognizer</module>
 
+        <!-- Personalizer -->
+        <module>sdk/personalizer/azure-ai-personalizer</module>
```
Comment (JimSuplizio) [inline pom.xml:84]: The only entry you should need in this file is the module entry down past line 900. @srnagar is in the process of removing these other entries for Jacoco/Javadoc etc. as they were originally in there for rollup reports. These files are only supposed to be aggregate pom files. You shouldn't need to add anything to this file other than the module entry below.

File: sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java (hunk 1 above)
Comment (sharathmalladi) [reply sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java:172]: This was required since in order to fix the spotcheck error: [ERROR] Low: The class com.azure.ai.personalizer.implementation.ModelsImpl$1 could be refactored into a named _static_ inner class [com.azure.ai.personalizer.implementation.ModelsImpl] At ModelsImpl.java:[line 167] SIC_INNER_SHOULD_BE_STATIC_ANON

File: pom.xml (hunk 2 above)
Comment (sharathmalladi) [reply pom.xml]: Fixed.

File: sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java (hunk 1 above)
Comment (sharathmalladi) [reply sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/implementation/ModelsImpl.java:172]: Will add a spot check exception instead as per recommendation.

**Review by samvaity — APPROVED** [review]
